types/ysdk/package.json

```
@@ -0,0 +1,19 @@
+{
+    "private": true,
+    "name": "@types/ysdk",
+    "version": "1.0.9999",
+    "nonNpm": true,
+    "nonNpmDescription": "TypeScript type definitions for Yandex.Games SDK",
+    "projects": [
+        "https://yandex.ru/dev/games/doc/en/"
+    ],
+    "devDependencies": {
+        "@types/ysdk": "workspace:."
+    },
+    "owners": [
+        {
+            "name": "koteelok",
+            "githubUsername": "yandex-games-plugins"
```

```suggestion
            "name": "Musa Asukhanov",
            "githubUsername": "koteelok"
```
an organisation cannot be owner (requires user). From Readme:
> DT has the concept of "Definition Owners" which are people who want to maintain the quality of a particular module's types.
Merging this to avoid nitpicking, you can take this by deciding if it's ok to merge